tools/dune

```diff
@@ -41,8 +41,8 @@
 (ocamllex coqwc)
 
 (executables
- (names coq_tex coq_dune)
- (public_names coq-tex coq_dune)
+ (names coq_tex coq_dune gen_install)
+ (public_names coq-tex coq_dune gen_install)
  (package coq)
```

Comment:
Shouldn't this particular executable be part of the `coq-refman` package instead? And don't you want to put it in `doc/tools`, or even `doc/tools/coqrst` or did you put it here and in the main package because you expected that it would be useful for other applications?
Another question: shouldn't this executable be private? IIUC the default is to install it and it doesn't really make sense to me that it be the case.

Reply:
Well spotted, that should be 100% private; sorry for the bug, fixing.